.github/workflows/greentea_cmake.yml

```diff
@@ -0,0 +1,30 @@
+name: test building greentea tests with cmake
+
+on: [pull_request]
+
+jobs:
+  build-greentea-cmake:
+    runs-on: ubuntu-latest
+    container: mbedos/mbed-os-env:latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install the latest mbed-tools
+        run: |
+            pip3 install --upgrade mbed-tools
```

Comment:
I added this because mbed-tools in the Docker image was outdated and GitHub Actions failed. Discussion https://github.com/ARMmbed/mbed-os/pull/14892#issuecomment-891006417 and https://github.com/ARMmbed/mbed-os/pull/14892#issuecomment-891122643. This is to ensure we have the latest released version.

Reply:
It looks like the latest Docker image is still from a month ago: https://hub.docker.com/r/mbedos/mbed-os-env/tags?page=1&ordering=last_updated

Reply:
I suggest leaving this in because the mbed-os-env doesn't have a working version.

Reply:
Could we file an issue to remove this? mbed-os-env really ought to have a working version

Reply:
https://github.com/ARMmbed/mbed-os/issues/14992

Reply:
We were using the wrong docker image (see Saheer's comment below) - now fixed.